kodeverk/src/main/java/no/nav/ung/kodeverk/dokument/Brevkode.java

```diff
@@ -93,6 +93,8 @@ public class Brevkode implements Kodeverdi {
     public static final Brevkode UNGDOMSYTELSE_SOKNAD = new Brevkode(UNGDOMSYTELSE_SOKNAD_KODE, "UNG Søknad", SØKNAD_RANGERING);
     public static final String UNGDOMSYTELSE_INNTEKTRAPPORTERING_KODE = "UNGDOMSYTELSE_INNTEKTRAPPORTERING";
     public static final Brevkode UNGDOMSYTELSE_INNTEKTRAPPORTERING = new Brevkode(UNGDOMSYTELSE_INNTEKTRAPPORTERING_KODE, "UNG Inntektrapportering", INNTEKTRAPPORTERING_RANGERING);
+    public static final String UNGDOMSYTELSE_BEKREFTELSE_PERIODEENDRING_KODE = "UNGDOMSYTELSE_BEKREFTELSE_PERIODEENDRING";
+    public static final Brevkode UNGDOMSYTELSE_BEKREFTELSE_PERIODEENDRING = new Brevkode(UNGDOMSYTELSE_BEKREFTELSE_PERIODEENDRING_KODE, "UNG Bekreftelse av periodeendring", VEDLEGG_RANGERING);
```

Comment:
Vi journalfører på `UNG Oppgavebekreftelse` for oppgaver fra brukerdialog.
Tenker at `UNG Bekreftelse av periodeendring` blir for spesifikk til endring av perioder. Dersom det er andre oppgver som skal løses på sikt kan dette skape forvirring.

Reply:
Men vi ønsker vel då ulike brevkoder for dei andre oppgavene?

Reply:
Eg endret til UNG Oppgavebekreftelse no